### ISSUE TITLE
Lesson 01 and 04: matplotlib.pyplot.show ignore any arguments

### ISSUE THREAD

**Issue by cmd-ntrf** [swcarpentry/python-novice-inflammation#173]
In lesson 01-numpy and 04-files, we are teaching that `matplotlib.pyplot.show` take in arguments either graphics or figures. For example:
```
ave_inflammation = data.mean(axis=0)
ave_plot = matplotlib.pyplot.plot(ave_inflammation)
matplotlib.pyplot.show(ave_plot)
```

If we refer to [matplotlib API](http://matplotlib.org/devdocs/api/pyplot_api.html#matplotlib.pyplot.show), there are no mentions that `show` takes matplotlib object as arguments. The stated behaviour is that `show` will display all figures and blocks.

I agree that it would be useful to be able to tell matplotlib which graphic we want to display, but the API as it is now does not support it. To convince ourselves, simply run the following code:
```
import numpy

data = numpy.loadtxt(fname='inflammation-01.csv', delimiter=',')

import matplotlib.pyplot

ave_inflammation = data.mean(axis=0)
ave_plot = matplotlib.pyplot.plot(ave_inflammation)
max_plot = matplotlib.pyplot.plot(data.max(axis=0))
min_plot = matplotlib.pyplot.plot(data.min(axis=0))
matplotlib.pyplot.show(ave_plot)
```

What we would expect is to only see the average plot, while we end up with all three graphics on the same figure.

From my point of view, since the arguments pass to show are simply drop, it does not serve any purpose to tell students to pass graphics or figure to show, and should simply call show without arguments in the courses.

**Comment by compilerguru** [comment]
Thanks very much for the good advice. I also have Mathematica and CDF Pro Player. I am asking Wolfram if there is a way  I can use these tools with DreamWeaver and Photoshop to execute python expressions which are then trivial to plot. Dynamically!

I love PyCharm and Pycharm Edu. Great for teaching, and managing your course assets while not letting coding in the way of more substantial concepts.

Thanks again
Cheers,

Patrick

**Comment by QuLogic** [comment]
This was fixed in #222.
